mongodbatlas/resource_mongodbatlas_cloud_backup_schedule.go

```diff
@@ -60,6 +60,42 @@ func resourceMongoDBAtlasCloudBackupSchedule() *schema.Resource {
 				Optional: true,
 				Computed: true,
 			},
+			"copy_settings": {
```

Comment:
@martinstibbe can you double check that we are also supporting ""deleteCopiedBackups" parameters as part of this update? https://www.mongodb.com/docs/atlas/reference/api-resources-spec/#tag/Cloud-Backup-Schedule/operation/updateCloudBackupBackupPolicyForOneCluster

Reply:
@martinstibbe are you planning to add this additional parameter in?

Reply:
ok thanks. can you call this out in docs explicitly then (just to get ahead as i'm sure customers are going to ask in future) that "deleteCopiedBackups" is not supported in TF and if they need can leverage instead in the Atlas Admin API or AtlasCLI?

Reply:
Added Note to docs

Reply:
Looks good!